src/Tools/ConfigurationSchemaGenerator/ConfigurationSchemaGenerator.csproj

```diff
@@ -20,8 +20,13 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" />
     <PackageReference Include="System.CommandLine" />
+    <PackageReference Include="IgnoresAccessChecksToGenerator" PrivateAssets="all" />
```

Comment:
I don't think we should be using this. We should only be using public APIs.

Which internal APIs are you using here?

Reply:
`Microsoft.CodeAnalysis.Shared.Extensions.ISymbolExtensions.GetDocumentationComment`, this is explained in https://github.com/dotnet/aspire/pull/4441/commits/218d43de70c75637b767198d6211f1e11828466f.

Reply:
I removed this and just used regular reflection.